ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -898,6 +898,7 @@ tests:
       E2E_RUN_TAGS: '@aws-ipi and @osd_ccs'
       OPENSHIFT_VERSION: "4.16"
       TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;OSD_CCS&
+      TEST_FILTERS_SUPPLEMENTARY: ~ChkUpgrade&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;OSD_CCS&
```

Comment:
I found these conditions have been covered in the `TEST_FILTERS`, not sure why still need this `TEST_FILTERS_SUPPLEMENTARY`?

Reply:
got the meaning after discussion with Zhaohua in slack, so no question from me. LGTM